utils/forest_shim/src/consts.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::address::Address;
+
+pub const SYSTEM_ADDR: Address = Address::new_id(0);
+pub const CRON_ADDR: Address = Address::new_id(3);
+pub const REWARD_ADDR: Address = Address::new_id(2);
```

Comment:
Let's copy all of them or none of them. And you can put them in the address namespace. Eg: `Address::SYSTEM_ACTOR`.

Reply:
okay

Reply:
fixed